frontend/src/features/maps/routes/MapCreateForm.tsx

```diff
@@ -127,6 +127,20 @@ export default function MapCreateForm() {
       privacy: mapInput.privacy,
       description: mapInput.description,
       location: !Number.isNaN(mapInput.location.latitude) ? mapInput.location : undefined,
+      // TODO: implement selector
+      geometry: {
+        rings: [
```

Comment:
I actually don't know. I didn't find documentation in PostGIS (although I haven't looked very much). All I did was build a `Polygon<Point>` struct in Rust and deserialize that to see what it would like in JSON. This was the result and it works.